docs/running/cloud/kubernetes.rst

```diff
@@ -369,6 +369,45 @@ Here is an example of running our test workflow leader locally, outside of Kuber
          --realTimeLogging \
          --logInfo
 
+Running CWL Workflows
+^^^^^^^^^^^^^^^^^^^^^
+
+Running CWL workflows on Kubernetes can be challenging, because executing CWL can require Toil to orchestrate containers of its own, within a Kubernetes job running in the Toil appliance container.
+
+Normally, running a CWL workflow should Just Work, as long as the workflow's Docker containers are able to be executed with Singularity, your Kubernetes cluster does not impose extra capability-based confinement (i.e. SELinux, AppArmor) that interferes with Singularity's use of user-mode namespaces, and you make sure to configure Toil so that its workers know where to store their data within the Kubernetes pods (which would be done for you if using a Toil-managed cluster). For example, you should be able to run a CWL workflow like this::
+
+   $ export TOIL_KUBERNETES_OWNER=demo-user  # This defaults to your local username if not set
+   $ export TOIL_AWS_SECRET_NAME=aws-credentials
+   $ export TOIL_KUBERNETES_HOST_PATH=/data/scratch
+   $ virtualenv --python python3 --system-site-packages venv
+   $ . venv/bin/activate
+   $ pip install toil[kubernetes,cwl]==5.8.0
+   $ toil-cwl-runner  \
+        --jobStore  aws:us-west-2:demouser-toil-test-jobstore \
+        --batchSystem kubernetes \
+        --realTimeLogging \
+        --logInfo \
+        --disableCaching \
+        path/to/cwl/workflow \
+        path/to/cwl/input/object
+        
+Additional ``cwltool`` options that your workflow might require, such as ``--no-match-user``, can be passed to ``toil-cwl-runner``, which inherits most ``cwltool`` options.
+
+AppArmor and Singularity
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Kubernetes clusters based on Ubuntu hosts will have AppArmor enabled on the host. AppArmor is a capability-based security enhancement system that integrates with the Linux kernel to enforce lists of things which programs may or may not do, called **profiles**. For example, an AppArmor profile could be applied to a web server process to stop it from using the ``mount()`` system call to manipulate the filesystem, because it has no business doing that under normal circumstances but might attempt to do it if compromised by hackers.
```

Comment:
```suggestion
Kubernetes clusters based on Ubuntu hosts often will have AppArmor enabled on the host. AppArmor is a capability-based security enhancement system that integrates with the Linux kernel to enforce lists of things which programs may or may not do, called **profiles**. For example, an AppArmor profile could be applied to a web server process to stop it from using the ``mount()`` system call to manipulate the filesystem, because it has no business doing that under normal circumstances but might attempt to do it if compromised by hackers.
```